src/Uno.UI.RemoteControl.Host/Uno.UI.RemoteControl.Host.csproj

```diff
@@ -34,6 +34,20 @@
 			<UndefineProperties>TargetFramework</UndefineProperties>
 		</ProjectReference>
 	</ItemGroup>
+	
+			<ItemGroup>
```

Comment:
@dr1rrb Let's add a paragraph on why this is needed.

Reply:
```suggestion
			<ItemGroup>
				<!-- 
					This forces the .net version to use, so the dev server will load the right core assemblies.
					Even if we make sure to run the version from the right .net<7|8> directory, if the computer has a newer version of .net than the version used to compiled the target app, 
					it might use that newer version (even if we have a RollFoward policy).
					This works only when the dev-server is started in the dll file directory! (Which is the case of hot-reload tests helper of the runtime-tests engine).
				-->
```